articles/relnotes/0.3-migration.md

```diff
@@ -166,7 +166,7 @@ To minimize the disruption to existing code that relied on this bug, we have add
 The following example illustrates how to use the `ToOperation` function to change a function `Square` to an operation `op`: 
```

Comment:
The function `ToOperation` has been renamed to `AsOperation` as a part of the general [Q# style guide](https://docs.microsoft.com/quantum/contributing/style-guide), and has been moved to the `Microsoft.Quantum.Convert` namespace (`xref:microsoft.quantum.convert` as a DocFX link).
```suggestion
The following example illustrates how to use the `AsOperation` function to change a function `Square` to an operation `op`: 
```